packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -116,7 +116,7 @@ export const LivenessCameraModule = (
   const isFaceMovementChallenge =
     useLivenessSelector(selectChallengeType) === FACE_MOVEMENT_CHALLENGE.type;
   const allowSelectableDevicesMobile =
-    !isMobileScreen || (isMobileScreen && isFaceMovementChallenge);
+    !isMobileScreen || isFaceMovementChallenge;
```

Comment:
If I'm reading this right, this will allow all cameras on all desktop for any challenge, and then for  faceMovementChallenge for desktop or mobile. Is that correct?

Reply:
There isn't a unit test specifically for camera selection, i can add it